sklearn/ensemble/_stacking.py

```diff
@@ -30,14 +30,15 @@
 from ..utils.multiclass import check_classification_targets
 from ..utils.validation import check_is_fitted
 from ..utils.validation import column_or_1d
+from ..utils.validation import _deprecate_positional_args
 
 
 class _BaseStacking(TransformerMixin, _BaseHeterogeneousEnsemble,
                     metaclass=ABCMeta):
     """Base class for stacking method."""
 
     @abstractmethod
-    def __init__(self, estimators, final_estimator=None, cv=None,
+    def __init__(self, estimators, *, final_estimator=None, cv=None,
```

Comment:
final_estimator should always be supplied, right? The default seems like a fallback, not sure why we even have it. Not a strong opinion though.